Update troubleshooting in README.md (#198)
leikoilja/ha-google-home: README.md

   3. Paste the password into the **Google account app password** field.
   4. Click **Submit**.

### Devices found, but not initialized

If the debug logs list your devices, but then show `Successfully initialized 0 Google Home devices` make sure you are logged into the correct Google account.

> Your account has to be able to access the devices in your Home.
> If unsure, you can check what account you are using in the _Google Home_ app.

Reviewer: ```suggestion
The account you are using with the integration must have access to your google devices.
If unsure, you can check what account and account permissions in the mobile _Google Home_ app.
```

Reviewer 2: @leikoilja Sorry, I think the second part became worse :)

How about
```markdown
If unsure, please check what account you are using in the _Google Home_ app and if you have these devices listed there.
```

Reviewer: haha sound good, thanks for the spot

Author: Yup, this sounds the best

Author: Slightly change it, what do you guys think?

> If unsure, please check what account you are using in the mobile _Google Home_ app and if your devices are listed there.

Reviewer 2: ```suggestion
If unsure, please check what account you are using in the _Google Home_ app and if your devices are listed there.
```
